src/mesh/voxel_cloud.rs

```diff
@@ -12,6 +14,1082 @@ use crate::plane::Plane;
 
 use super::{primitive, tools, Face, Mesh, NormalStrategy};
 
+/// Row in the marching cubes lookup table.
```

Comment:
Can we please move all the marching cubes implementation details below the public items so they don't clutter the start of the file?

Reply:
possibly yes, i'll have to try